experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -136,6 +158,37 @@ export class FieldProxyTarget extends ProxyTarget<FieldAnchor> implements Editab
 		return this.cursor.getFieldLength();
 	}
 
+	/* eslint-disable no-param-reassign */
+
+	public slice(start?: number, end?: number) {
+		const length = this.length;
+
+		start = normalizeIndex(start, length);
+		if (start < 0) start = 0;
+
+		if (end === undefined) {
+			end = length;
+		} else {
+			end = normalizeIndex(end, length);
+			if (end < 0) end = 0;
+		}
```

Comment:
Will document the special pre/post steps for normalization of start/end.  (It's all nicely described on MDN.)